packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -90,7 +86,7 @@ def jasmine_node_test(
     nodejs_test(
         name = name,
         data = all_data,
-        entry_point = jasmine_entry_point,
+        entry_point = coverage_entry_point,
```

Comment:
Assuming we can't tell in the macro context that coverage is enabled to select `jasmine_entry_point` if its not?

Reply:
we can only ttell that we're working under coverage with either an `env` var or access to `ctx` neither of which we have in this macro, not sure how else to pick what entry point to use here